packages/xo-server/src/_MultiKeyMap.js

```diff
@@ -1,58 +1,88 @@
-const kValue = Symbol('value')
+class Node {
+  constructor(value) {
+    this.children = new Map()
+    this.value = value
+  }
+}
 
-function del(map, i, n, keys) {
+function del(node, i, n, keys) {
```

Comment:
Do we need `n`? Can't we work with `keys.length`?

Reply:
Minor opt, I'll remove it.